src/backup.js

```diff
@@ -164,7 +164,12 @@ function zipFiles (config, tar, bucket, files) {
   return tar.zipFiles(dataPath, list, archive)
 }
 
-module.exports = function (config, glob, tar, bucket) {
+module.exports = function (
+    config = require('./config'),
+    glob = require('globulesce'),
+    tar = require('./tar'),
+    bucket = require('./bucket')
+  ) {
```

Comment:
This might be a bit contentious, but it makes the DI style clearer -- it's easier to see what these injected modules are supposed to be, while still letting you override them.

Reply:
personally, i love this. much easier to follow at a glance